code/game/objects/items/devices/radio/headset.dm

```diff
@@ -93,7 +98,8 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 
 /obj/item/radio/headset/syndicate/alt/ComponentInitialize()
 	. = ..()
-	AddComponent(/datum/component/wearertargeting/earprotection, list(ITEM_SLOT_EARS))
+	if(!HAS_TRAIT(SSstation, STATION_TRAIT_NEW_UNIFORM_STANDARDS))
+		AddComponent(/datum/component/wearertargeting/earprotection, list(ITEM_SLOT_EARS))
```

Comment:
This makes me very uncomfortable

Reply:
they're no longer headsets on the ear, so

Reply:
I know, but I really don't like it